packages/kit/test/prerendering/basics/test/tests.spec.js

```diff
@@ -240,7 +240,7 @@ test('prerenders a page in a (group)', () => {
 
 test('injects relative service worker', () => {
 	const content = read('index.html');
-	expect(content).toMatch("navigator.serviceWorker.register('./service-worker.js')");
+	expect(content).toMatch("navigator.serviceWorker.register('./service-worker.js'");
```

Comment:
Removed the parentheses at the end so that this test passes whether it's `('./service-worker.js', { type: 'module' })` in dev or `('./service-worker.js')` in prod